model/Hierarchy.php

```diff
@@ -641,9 +646,16 @@ public function liveChildren($showAll = false, $onlyDeletedFromStage = false) {
 		}
 
 		$baseClass = ClassInfo::baseDataClass($this->owner->class);
+
+		$hidden = array();
+		if ($baseClass::config()->get('hideFromHierarchy')) {
```

Comment:
To be sneaky and/or lazy, if you add the config I suggested above you can do `$this->owner->config()->hide_from_hierarchy` and you don't need to check to see if the config exists.